cosmos/milkyway.json

```diff
@@ -0,0 +1,114 @@
+  },
+  "stakeCurrency": {
+    "coinDenom": "MILK",
+    "coinMinimalDenom": "umilk",
+    "coinDecimals": 6,
+    "coinGeckoId": "",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/milkyway/chain.png"
+  },
+  "walletUrl": "https://beta.milkyway.zone/",
+  "walletUrlForStaking": "",
+  "bip44": {
+    "coinType": 118
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "milk",
+    "bech32PrefixAccPub": "milkpub",
+    "bech32PrefixValAddr": "milkvaloper",
+    "bech32PrefixValPub": "milkvaloperpub",
+    "bech32PrefixConsAddr": "milkvalcons",
+    "bech32PrefixConsPub": "milkvalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "TIA",
+      "coinMinimalDenom": "ibc/6C349F0EB135C5FA99301758F35B87DB88403D690E5E314AB080401FEE4066E5",
+      "coinDecimals": 6,
+      "coinGeckoId": "celestia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/celestia/chain.png"
+    },
+    {
+      "coinDenom": "milkTIA",
+      "coinMinimalDenom": "ibc/16065EE5282C5217685C8F084FC44864C25C706AC37356B0D62811D50B96920F",
+      "coinDecimals": 6,
+      "coinGeckoId": "milkyway-staked-tia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/osmosis/factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/milktia.png"
+    },
+    {
+      "coinDenom": "stTIA",
+      "coinMinimalDenom": "ibc/8D4FC51F696E03711B9B37A5787FB89BD2DDBAF788813478B002D552A12F9157",
+      "coinDecimals": 6,
+      "coinGeckoId": "stride-staked-tia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/sttia.png"
+    },
+    {
+      "coinDenom": "Locked TIA",
+      "coinMinimalDenom": "locked/ibc/6C349F0EB135C5FA99301758F35B87DB88403D690E5E314AB080401FEE4066E5",
+      "coinDecimals": 6,
+      "coinGeckoId": "celestia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/celestia/chain.png"
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "TIA",
+      "coinMinimalDenom": "ibc/F1183DB3D428313A6FD329DF18219F9D6B83257D07D292EA9EC1D877E89EC2B0",
+      "coinDecimals": 6,
+      "coinGeckoId": "celestia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/celestia/chain.png",
+      "gasPriceStep": {
+        "low": 0.0025,
+        "average": 0.003,
+        "high": 0.004
+      }
+    },
+    {
+      "coinDenom": "TIA",
+      "coinMinimalDenom": "ibc/6C349F0EB135C5FA99301758F35B87DB88403D690E5E314AB080401FEE4066E5",
+      "coinDecimals": 6,
+      "coinGeckoId": "celestia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/celestia/chain.png",
+      "gasPriceStep": {
+        "low": 0.0025,
+        "average": 0.003,
+        "high": 0.004
+      }
+    },
```

Comment:
duplicated

Reply:
This is actually not duplicated, we simply have two versions of TIA supported inside our chain: one that is bridged through Osmosis, and the other ones that is bridged through Celestia directly. We support both as our fee tokens.